profiles/sink/Sink_HDMIIN.yaml

```diff
@@ -20,3 +20,8 @@ dsHdmiIn:
   gameFeatures:
     feature: "allm"
     count: 1
+  # HDMI_COMPATIBILITY_VERSION_14 = 0x00, /*!< Hdmi Compatibility Version 1.4 */
+  # HDMI_COMPATIBILITY_VERSION_20 = 0x01,     /*!< Hdmi Compatibility Version 2.0 */
+  # HDMI_COMPATIBILITY_VERSION_21 = 0x02,     /*!< Hdmi Compatibility Version 2.1 */
+  # HDMI_COMPATIBILITY_VERSION_MAX = 0x03     /*!< Out of bounds */
+  HdmiCompatibilityVersion: 0x01
```

Comment:
number of ports are 3 but the version goes with one value. should it be maintain in list based on the port number?

Reply:
updated to list and modified code